.github/workflows/export_category.yml

```diff
@@ -0,0 +1,62 @@
+name: Export Category 
+
+on:
+  workflow_dispatch:
+    inputs:
+      category:
+        description: 'Corresponds to tab of 1572-B data matrix '
+        required: true
+        default: housing_production
+
+
+jobs:
+  export:
+    runs-on: ubuntu-20.04
+    env:
+      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
+      
+    strategy:
+      matrix:
+        geography:
+          - puma
+          - borough
+          - citywide
+        category:
+          - housing_production
+          #- quality_of_life
```

Comment:
Do we want to keep the version on main with this commented out? I think the default should be that we export all categories, not a big deal to comment out though

Reply:
This is a good flag, the workflow is messy. It should be changed to either always do all 3 categories or allow the user to select just one. Do you want to implement this upgrade? If not I could

Reply:
This [data library workflow](https://github.com/NYCPlanning/db-data-library/blob/261-Image-Workflow/.github/workflows/update_image.yml) limits the user to a set of choices. That would be a good upgrade here. In the future we could implement an "all" option but that might be hard to achieve right now